js/src/sdk/client/types.gen.ts

```diff
@@ -1,20 +1,266 @@
 // This file is auto-generated by @hey-api/openapi-ts
 
 /**
- * Member information
+ * Team Member information
  */
 export type MemberInfoResDTO = {
+  /**
+   * The ID/UUID of the member.
+   */
   id: string;
-  projectId: string;
+  /**
+   * The ID/UUID of the organization to which the member belongs.
+   */
+  orgId: string;
+  /**
+   * The email address of the member.
+   */
   email: string;
+  /**
+   * The name of the member.
+   */
   name: string;
-  role: string;
-  metadata?: unknown;
+  /**
+   * The role of the member.
+   */
+  role: "admin" | "developer";
+  /**
+   * The metadata of the member. This store some additional information about the member.
+   */
+  metadata?: {
+    [key: string]: unknown;
+  };
+  /**
+   * The date and time when the member was created.
+   */
   createdAt: string;
+  /**
+   * The date and time when the member was last updated.
+   */
   updatedAt: string;
+  /**
+   * The date and time when the member was deleted.
+   */
   deletedAt?: string;
 };
 
+/**
+ * The role of the member.
+ */
+export type role = "admin" | "developer";
+
+export type HttpError = {
+  /**
+   * HTTP status code
+   */
+  status: number;
+  /**
+   * Error message
+   */
+  message: string;
+  /**
+   * Request ID, used for tracing the request. This is very helpful for internal teams to debug issues.
+   */
+  requestId: string;
+};
+
+export type BadRequestError = {
+  /**
+   * Additional arguments that caused the error
+   */
+  details: {
+    [key: string]: unknown;
+  };
+  /**
+   * The type of error
+   */
+  type: "BadRequestError";
+  /**
+   * HTTP status code
+   */
+  status: number;
+  /**
+   * Error message
+   */
+  message: string;
+  /**
+   * Request ID, used for tracing the request. This is very helpful for internal teams to debug issues.
+   */
+  requestId: string;
+};
+
+/**
+ * The type of error
+ */
+export type type = "BadRequestError";
```

Comment:
The `type` enum is defined multiple times with different values (type, type2, type3, type4, type5). This could lead to confusion and type safety issues. Consider:
1. Using more descriptive names for these types (e.g., `ErrorType`, `RequestBodyType`, etc.)
2. Consolidating similar enums where possible
3. Adding documentation to clarify the purpose of each type